private/views/index.hjs

```diff
@@ -96,7 +27,8 @@
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/react.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/JSXTransformer.js"></script>
-    <script src="/javascripts/components/footer.js" type="text/jsx"></script>
+    <script src="../javascripts/components/footer.js" type="text/jsx"></script>
+    <script src="../javascripts/components/header.js" type="text/jsx"></script>
```

Comment:
index is found at the root level (e.g. localhost/). No need to go one step back.